src/meadowrun/run_job.py

```diff
@@ -487,6 +487,7 @@ async def mirror_local(
         interpreter: Union[
             LocalInterpreter, InterpreterSpecFile, ContainerInterpreterBase, None
         ] = None,
+        non_py_files_globs: Union[str, Iterable[str], None] = None,
```

Comment:
@kurtschelfthout I'm not sure how I would extend this to files that aren't in the current working directory. A couple options:

1. The current (undocumented) behavior--you can provide a glob here that has a relative/absolute path outside of the current working directory to specify non-.py file. Main weirdness here is that you can glob-specify something that isn't on a python path or a non-python path, and it will just get ignored.
2. We could add a parameter like `non_python_paths` where you could specify tuples of additional non-python paths along with globs specific to each of those python paths. The weirdness there would be that you couldn't specify non *.py files in your python paths. That's probably a pretty rare case, but it's possible there's some python code using `__file__` to  reference e.g. a txt file in the same directory.

I'm in favor of just leaving the behavior as is, I think the current documented behavior should cover 80% of the use cases, but worth just thinking ahead a little bit.